processor/src/lib.rs

```diff
@@ -14,6 +14,7 @@ mod operations;
 
 mod system;
 use system::System;
+pub use system::FMP_MIN;
```

Comment:
We should move this constant into the `core` create - but probably in another PR.

Reply:
Sure. Before refactoring the aux table there were quite a few constants needed for tests that I didn't feel really belonged in core (Hasher & Bitwise internals), so I ended up not moving any of them over. I agree I could see this one going over, but I'll leave it for the future.